presto-geospatial/src/main/java/com/facebook/presto/geospatial/BingTileFunctions.java

```diff
@@ -217,7 +217,7 @@ private static void checkGeometryToBingTilesLimits(OGCGeometry geometry, boolean
             checkCondition(tileCount <= 1_000_000, "The number of input tiles is too large (more than 1M) to compute a set of covering bing tiles.");
         }
         else {
-            long complexity = tileCount * getPointCount(geometry.getEsriGeometry());
```

Comment:
Change the commit title to say "Fix potential overflow in Bing tile functions".

Also, for better readability, add parenthesis around `(long) tileCount`.